src/DynamoCore/DynamoCore.csproj

```diff
@@ -78,9 +78,8 @@ limitations under the License.
     <Reference Include="Analytics.NET.Google">
       <HintPath>..\..\extern\Analytics.NET\Analytics.NET.Google.dll</HintPath>
     </Reference>
-    <Reference Include="Greg, Version=1.0.6176.18754, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Greg.1.0.6176.18754\lib\net45\Greg.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="Greg, Version=1.1.6905.23659, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Greg.1.1.6905.23659\lib\net45\Greg.dll</HintPath>
```

Comment:
@mjkkirschner should this package be in a `\lib\net47\Greg.dll` location now?

Reply:
nope, we have not updated greg.dll to net 47 yet.